### PR TITLE
Add information on filtering tests

### PR DESCRIPTION
**Pull request by jrjohnson**
As suggested in https://github.com/ember-cli/ember-cli/pull/6036 this information belongs in the guides as opposed to the command help docs because it is QUnit specific and that is the happy path for ember-cli tests.

The part about inverting tests with a '!' is particularly useful and hard to find elsewhere.

### CODE REVIEW

**Comment by ballPointPenguin** [comment]
This is helpful, thank you.

File: source/localizable/testing/index.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ Tests run using `ember server` have the environment configuration `development`,
 
 These commands run your tests using [Testem] to make testing multiple browsers very easy. You can configure Testem using the `testem.js` file in your application root.
 
+#### Choosing the Tests to Run
+
+To run a subset of your tests by title use the `--filter` option.  Quickly test your current work `ember test --filter="dashboard"` or only run a certain type of test `ember test --filter="integration"`. When using QUnit (the default) it is possible to exclude tests by adding an exclamation point to the beginning of the filter `ember test --filter="!acceptance"`.
```
Comment (locks) [inline source/localizable/testing/index.md]: comma before "or only run".
missing newline after period.
can you remove "(the default)"? we already allude to that fact above.

**Comment by locks** [comment]
@jrjohnson left you some nitpicks inline, thanks for the contribution :)

**Comment by jrjohnson** [comment]
@locks no problem, happy to help. I made the changes requested.

**Pull request merged**